src/world/mod.rs

```diff
@@ -137,37 +136,46 @@ impl World {
     }
 
     pub fn update_block(&self, pos: Position) {
-        for yy in -1..2 {
-            for zz in -1..2 {
-                for xx in -1..2 {
-                    let bp = pos + (xx, yy, zz);
-                    let current = self.get_block(bp);
-                    let new = current.update_state(self, bp);
-                    if current != new {
-                        self.set_block_raw(bp, new);
+        if self.protocol_version < 404 {
```

Comment:
why don't we do this in later versions?

Reply:
Could you add a comment explaining this rationale?

Reply:
I've updated this